app/services/coronavirus/pages/content_builder.rb

```diff
@@ -73,6 +72,8 @@ def required_hub_page_keys
 
     def github_raw_data
       YamlFetcher.new(page.raw_content_url).body_as_hash
+    rescue RestClient::Exception
+      raise GitHubConnectionError
```

Comment:
I notice no tests needed to be changed for this, which makes me wonder if this is untested code (unless it's covered at a high level)? 

Same thought on the next commit too.